templates/configuration/graphdb-node-configmap.yaml

```diff
@@ -11,6 +11,7 @@ data:
     -Dentity-pool-implementation=transactional
     -Dhealth.max.query.time.seconds=60
     -Dgraphdb.vhosts={{ $.Values.deployment.protocol }}://{{ include "resolveDeploymentHost" $ }}{{ $.Values.graphdb.workbench.subpath }}
+    -Dgraphdb.external-url={{ $.Values.deployment.protocol }}://{{ include "resolveDeploymentHost" $ }}{{ $.Values.graphdb.workbench.subpath }}
```

Comment:
This line absolutely breaks the cluster communication.
Can you double check that it's required for a single node, since in my testing its not. I wanna make sure its not a "it works on my machine" situation. 
If it's required maybe put it in an if for ONLY single node deployments?

Reply:
it's required for explicit transactions and when the context is non root e.g. POST /graphdb/repositories/test/transactions will return location header without the context /graphdb